thetis/solver.py

```diff
@@ -727,8 +731,13 @@ def add_callback(self, callback, eval_interval='export'):
 
     def export(self):
         self.callbacks.evaluate(mode='export')
+        # set uv to total uv instead of deviation from depth average
+        # TODO find a cleaner way of doing this ...
```

Comment:
Yeah, would it not be cleaner to have `uv_3d` always contain the total velocity and have a separate `uv_dev_3d` and pass that into the relevant places. Not sure how much re-engineering that requires...